### ISSUE TITLE
Loading a Waters raw file in MZmine using Wine

### ISSUE THREAD

**Issue by gtamayo59** [mzmine/mzmine2#385]
Hi there! I must say first that programming is not one of my strengths... so, could you provide a simple guide for those using Mac OS X, to successfully open Waters raw files in MZmine?

I downloaded the Wine API as you suggested, downloaded the winehq-stable package that made me download another APIs as well.... but then in MZmine when trying to open the file, the same message is displayed (see attached png), as if I haven't installed Wine.

The log messages:
[18:07:55|INFO|NativeFileReadTask]: Opening file /Users/giselle/Desktop/untitled folder/35-1-070515-1_08052015_002.raw
java.io.IOException: Cannot run program "wine": error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at java.lang.Runtime.exec(Runtime.java:620)
	at java.lang.Runtime.exec(Runtime.java:485)
	at net.sf.mzmine.modules.rawdatamethods.rawdataimport.fileformats.NativeFileReadTask.run(NativeFileReadTask.java:132)
	at net.sf.mzmine.taskcontrol.impl.WorkerThread.run(WorkerThread.java:61)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:248)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 4 more
[18:07:55|SEVERE|WorkerThread]: Error of task Opening file /Users/giselle/Desktop/untitled folder/35-1-070515-1_08052015_002.raw: java.io.IOException: Cannot run program "wine": error=2, No such file or directory (ProcessBuilder.java:1048)

I know that wine is running, but I don't know if it has to be placed in certain location. 

So:  MZmine version: 2.29
       OS version:  MacOS Yosemite, 10.10.5
       java version "1.8.0_45"
      Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
      Java HotSpot(TM) 64-Bit Server VM (build 25.45-b02, mixed mode)
       R version 3.3.1 (2016-06-21) -- "Bug in Your Hair"
      Copyright (C) 2016 The R Foundation for Statistical Computing
      Platform: x86_64-apple-darwin13.4.0 (64-bit)
      ![screen shot 2017-11-15 at 18 14 09](https://user-         images.githubusercontent.com/33704903/32867114-d7587148-ca30-11e7-9051-1cd9f142f1d8.png)

My computer is: iMac, Late2013, 32 GB RAM, 3.4 GHz Intel core i5, plenty HD space

Thanks in advance!

**Comment by tomas-pluskal** [comment]
First try running the watersrawdump.exe program manually from terminal using wine. If you make that work, the rest should be easy.

**Comment by gtamayo59** [comment]
Tomas, thanks for getting back to me. Where do I find the watersrawdump.exe file?

G.
Giselle Tamayo

E-mail personal: tamayo.castillo@gmail.com

**Comment by tomas-pluskal** [comment]
In the mzmine's lib\vendor_lib\waters folder

**Comment by gtamayo59** [comment]
Tomas:  I see. watersrawdump.exe is in MZmine. But in terminal there is a file named wine, that apparently doesn’t run properly. I should write in terminal:  wine watersrawdump.exe and it should execute, right? Well, it doesn’t. I guess that I did not installed properly wine?

Should I have installed in another location? Being in terminal, the list of applications is the following:




Giselle Tamayo

E-mail personal: tamayo.castillo@gmail.com
